src/Microdown/MicParagraphBlock.class.st

```diff
@@ -30,7 +30,7 @@ MicParagraphBlock >> addLineAndReturnNextNode: line [
 
 	text := text
 		ifNil: [ line trim ]
-		ifNotNil: [ text , String cr , line trim ].
+		ifNotNil: [ text , String space , line trim ].
```

Comment:
I am not happy about this thing happening in the parser, as it throws out information. 
In github markdown, new lines in input are respected, and I think one need very good reasons to make it impossible to make a renderer which mirrors github output.

Reply:
Rather I could imagine composers to merge lines if needed. I believe it is the visitText: methods in MicRichTextComposer which need to remove cr's. This could be done as:
```smalltalk
visitText: anInlineBlock
	canvas << (anInlineBlock substring asText copyReplace: String cr with: textStyler intextLinemerger)
```
moving the actual behaviour to the text styler which can be set before the composer starts to compose.

For the latex version I guess something similar can be done.